src/components/FrontPage/TeacherInfobox.scss

```diff
@@ -11,10 +11,15 @@
   font-weight: 300;
 }
 
+.infoBoxContainer {
+  display: flex;
```

Comment:
Jeg ser ingen forskjell når jeg fjerner `display: flex` (eller hele styles.infoBoxContainer for den saks skyld). Trenger vi den?

(Forøvrig, jeg har merket at når jeg kjører flex i IE11 og eldre, så har ikke den støtte for flexbox. Jeg vet at det er brukt flexbox flere andre steder i codeclub-viewer, og jeg vet at IE11 og eldre er på vei ut, men vi snakker jo om at denne siden skal brukes av mange skoler som kjører gammelt IT-utstyr --- og det vet jeg, for jeg jobber på Utdanningsdirektoratet og har statistikk! --- og det hadde jo vært litt synd om elevene ser en ødelagt side bare fordi skolen ikke har råd til (eller vett til) å oppgradere... hadde det kanskje vært en ide å prøve å skrive om de andre stedene som bruker flexbox også? Ikke i denne PR'en selvsagt, måtte bli en egen issue i såfall, men tenkte jeg skulle stille spørsmålet...)